cristin-person/src/main/java/no/unit/nva/cristin/person/institution/update/PersonInstInfoPatchSerializer.java

```diff
@@ -0,0 +1,37 @@
+package no.unit.nva.cristin.person.institution.update;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import java.io.IOException;
+import java.util.Objects;
+import no.unit.nva.cristin.person.model.nva.PersonInstInfoPatch;
+
+public class PersonInstInfoPatchSerializer extends StdSerializer<PersonInstInfoPatch> {
+
+    public static final String EMAIL_FIELD = "email";
+    public static final String PHONE_FIELD = "phone";
+
+    public PersonInstInfoPatchSerializer() {
```

Comment:
PersonAffiliationPatchSerializer?

Reply:
There is some endpoints coming later related to a person's affiliations/employments. Names might crash into each other.